### ISSUE TITLE
Abstract: Bash script using KUWAC?

### ISSUE THREAD

**Issue by Kreyren** [PhoenicisOrg/phoenicis#1849]
Referenced on #1787 i understand that maintainance of bash scripts support for phoenicis is insane..

Can't we in theory use **finished** KUWAC (Krey's Universal WineApp Configurator) for this? https://github.com/RXT067/Scripts/tree/master/KUWAC

It's concept of a bash script that is able to make wineprefix configuration using bash backend and configuration alike https://github.com/RXT067/Scripts/blob/master/KUWAC/CONFIGURATION/LeagueOfLegends.sh for each wineprefix that is downloaded from upstream. Originally i was using it for installing leagueoflegends using gentoo's package manager: https://www.youtube.com/watch?v=Uvmsw4_25ZY&feature=youtu.be which worked on platinum based on provided info (note that performance is limited by recording and unoptimized sheduler, works on +-140 FPS using AMD7870).

Whole script has few kbs (megs max) and interpretation shoudn't be an issue assuming that POL4 is able to use console with bash. 

Concept is to maintain backend in phoenicis (willing to do that) for configuration to be used. Simmilar to making ebuilds.

**Comment by plata** [comment]
I don't see the advantage.

**Comment by Kreyren** [comment]
> I don't see the advantage.

Making scripts in phoenicis maintainable using bash not an advantage?

**Comment by qparis** [comment]
I have the feeling that we've already discussed about this before. Why opening a new ticket? 
What is the differences with POLv4 frankly?

**Comment by Kreyren** [comment]
@qparis i understand that we discussed that upstream support for bash is not sane, this is new issue about it's implementation using KUWAC-like bash that is able to handle phoenicis verbs. -> Seems like different issue to me.

I stated in https://github.com/PhoenicisOrg/scripts/issues/855#issuecomment-461371996 that:
> POL4 is unfinished garbage that lacks community, guidaince, Quality assurance, Acceptable documentation, core+mandatory functionality and the way it handles funtions .. that's why it doesn't have people that are making scripts on it based on my personal experience..

Assuming that phoenicis has console with bash binnary and selected commands i believe that this could be done better and with low maintainance for phoenicis assuming that no special functions to interact with GUI are not mandatory or in general allow invoking bash scripts using phoenicis simmilar to gentoo's ebuilds.
